docker/jenkins.Dockerfile

```diff
@@ -0,0 +1,49 @@
+# https://www.jenkins.io/doc/book/installing/docker/ - Follow these steps for starting jenkins.
+FROM nvidia/cuda:11.4.2-cudnn8-devel-ubuntu20.04
```

Comment:
ada-00 is using 18.04 right?

Reply:
It is the docker image so it does not matter.

Reply:
Any reason why you're using cuda 11.4?

Reply:
Nope. Didn't try others. The current GPU on ada-00 works with 11.4 so I went with it.

Reply:
The latest torchvision works best with cuda 11.6, I'm not entirely sure it works with 11.4. Can we try using 11.6 here for better compatibility?

Reply:
Done!